src/DynamoCoreWpf/Views/PackageManager/Controls/FilterTagControl.xaml.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace Dynamo.PackageManager.UI
+{
+    /// <summary>
+    /// Interaction logic for FilterTagControl.xaml
+    /// </summary>
+    public partial class FilterTagControl : UserControl
+    {
+        public string TagName
```

Comment:
Would you add comments for all the public members if they indeed needs to be public?

Reply:
Done and done!